docs/docs/atom-archive/getting-started/sections/atom-basics.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Atom Basics
+---
+
+::: danger STOP
+
+This is an archive of the atom documentation. This is only here for historical
+purposes. This is solely a functional reference of how things used to work.
+Processes in this section may have changed in both functionality and
+presentation since these were originally written. Our updated documentation can
+be found at: <insert link here>
+
+:::
+
+### Atom Basics
+
+Now that Atom is installed on your system, let's fire it up, configure it and get acquainted with the editor.
+
+When you launch Atom for the first time, you should get a screen that looks like this:
+
+![Atom's welcome screen](@images/atom/first-launch.png)
+
+This is the Atom welcome screen and gives you a pretty good starting point for how to get started with the editor.
+
+#### Terminology
+
+You can find definitions for all of the various terms that we use throughout the manual in our [Glossary](/resources/sections/glossary/).
+
+#### Command Palette
+
+In that welcome screen, we are introduced to probably the most important command in Atom, the Command Palette. If you press <kbd class="platform-mac">Cmd+Shift+P</kbd><kbd class="platform-windows platform-linux">Ctrl+Shift+P</kbd> while focused in an editor pane, the command palette will pop up.
```

Comment:
While it's valid to use HTML within Markdown, I find it problematic. It's ugly, but the benefit of using a `<kbd>` outweighs the disadvantages. But I think it's highly problematic to have HTML classes in a Markdown document. It's used for styling and as such should be no part of the content.